app/frontend/src/components/MarkdownInput/MarkdownInput.tsx

```diff
@@ -75,6 +80,31 @@ export default function MarkdownInput({
   const fieldOnChange = useRef<typeof field.onChange>(field.onChange);
 
   useEffect(() => {
+    const uploadButton = imageUpload ? document.createElement("button") : null;
+
+    if (imageUpload) {
+      uploadButton!.type = "button";
+      uploadButton!.innerHTML = "Pic";
+      uploadButton!.style.margin = "0";
+      uploadButton!.style.background = "transparent";
+      uploadButton!.addEventListener("click", () => {
+        setImageDialogOpen(true);
+      });
```

Comment:
I think we may want a `removeListener` in the `useEffect` cleanup function to remove this event handler to avoid memory leaks